nginx/Dockerfile

```diff
@@ -126,6 +126,5 @@ STOPSIGNAL SIGTERM
 # Enable a normal user to create new server keys off set CA
 RUN addgroup -g 1000 -S oqs && adduser --uid 1000 -S oqs -G oqs && chown -R oqs.oqs ${INSTALLDIR}
 USER oqs
-
+ENV OPENSSL_CONF ${OSSLDIR}/ssl/openssl.cnf
```

Comment:
This bypasses all `nginx` config logic. Why is this necessary? Shouldn't this be documented somewhere?